src/aks-preview/azext_aks_preview/vendored_sdks/kubernetes/.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+# ref: https://docs.gitlab.com/ee/ci/README.html
```

Comment:
I would say it's not necessary to include the sample, doc, test, ci related things in this vendored SDK.